### DISCUSSION TITLE
Envos and Venue: origin and meaning of the names

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1328] [General]
What's the origin and meaning of the names Envos and Venue, the Xerox spinoffs that distributed and supported Medley?

**Comment by masinter** [comment]
I don't know about Venue. There might be some clues in the emails in the Envos repo because it covered the transition.

I do remember a "brainstorming" session to name the spin-out -- we were following the suggestion that we wanted a name that was meaningless in common languages (the example of the chevy Nova in spanish was given) I remember suggesting "EnvOS" -- an Environment and also an Operating System.

**Comment by pamoroso** [comment]
> "EnvOS" -- an Environment and also an Operating System.

That's brilliant, and accurate.
